reactive-client/build.gradle

```diff
@@ -0,0 +1,57 @@
+buildscript {
+    repositories {
+        jcenter()
+        google()
+        maven {
+            url 'http://nexus.gluonhq.com/nexus/content/repositories/releases'
+        }
+    }
+    dependencies {
+        classpath 'org.javafxports:jfxmobile-plugin:2.0.28'
+    }
+}
+
+apply plugin: 'org.javafxports.jfxmobile'
+
+repositories {
+    jcenter()
+    maven {
+        url 'http://nexus.gluonhq.com/nexus/content/repositories/releases'
+    }
+}
+
+mainClassName = 'org.gluonhq.RxGluonApp'
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+dependencies {
+    if (JavaVersion.current() == JavaVersion.VERSION_1_9) {
```

Comment:
Check for JDK 9+ with `JavaVersion.current().isJava9Compatible()`

Reply:
Will do, but note that this was generated by Gluon idea plugin